src/main/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSource.java

```diff
@@ -203,7 +203,7 @@ public boolean accept(Path path) {
             }
             setHadoopBAMConfigurationProperties(filePath, referencePath);
             return SAMHeaderReader.readSAMHeaderFrom(path, ctx.hadoopConfiguration());
-        } catch (IOException e) {
+        } catch (Exception e) {
```

Comment:
@tomwhite Do we know what the actual exception type is here?  It's always a bit strange when you catch every exception.